thorlcr/activities/thactivityutil.cpp

```diff
@@ -403,7 +404,16 @@ void calcMetaInfoSize(ThorDataLinkMetaInfo &info, ThorDataLinkMetaInfo *infos, u
                 else
                     max = -1;
             }
+            if (NotFound == fastThroughVal)
+                fastThroughVal = prev.fastThrough ? 1 : 0;
+            else if ((prev.fastThrough?1:0) != fastThroughVal) // i.e. mismatch
+                fastThroughVal = 2; // mismatch
         }
+        if (0 == fastThroughVal)
```

Comment:
Shouldn't it be fast through unless there is an input that isn't fast through?  E.g., if there are no inputs it should be fast through, if there are mixed it should NOT be fast through.

Reply:
yes agree, there's a separate check for 0/1 parts at start of function, but yes should set fastThrough to true there if 0.
Should also set totalRowsMin and totalRowsMax to 0 there too.